pages/visitors/[id]/name.js

```diff
@@ -14,8 +14,12 @@ import propsWithContainer from "../../../src/middleware/propsWithContainer";
 import Error from "next/error";
 import Form from "../../../src/components/Form";
 import logger from "../../../logger";
+import { v4 as uuidv4 } from "uuid";
 
-const Name = ({ callId, error, callPassword }) => {
+const Name = ({ callId, error, callPassword, correlationId }) => {
+  useEffect(() => {
```

Comment:
I think my react is rusty but why do we need `useEffect` here?

Reply:
sorry plz ignore me ^ - it makes sense now that I have read the notes section of the PR description :)

Reply:
As far as I know its best practice to wait until react has updated the dom when you dont have code that executes as a result of a specific event in the page (like on onlick) but instead happens at some point during the lifecycle of the component. I think this is how I see this, that once the call has been ended ( i.e when this component mounts, which I think is the equivalent of use effect in hooks?), then the event logger would be called.